src/index.html

```diff
@@ -22,6 +22,336 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="logo">
+        <a
+          class="logo__link"
+          href="#"
+        >
+          <img
+            class="logo__img"
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
```

Comment:
The `data-qa` attribute is used for testing purposes and should not be present in the production code unless it's being used for automated testing.

Reply:
> The `data-qa` attribute is used for testing purposes and should not be present in the production code unless it's being used for automated testing.

it's being used for automated testing.